src/etc/inc/util.inc

```diff
@@ -45,9 +45,11 @@ function isvalidpid($pidfile) {
 
 function is_process_running($process) {
 	$output = "";
-	exec("/bin/pgrep -anx " . escapeshellarg($process), $output, $retval);
-
-	return (intval($retval) == 0);
+	if (empty($process)) {
```

Comment:
Should be `!empty()`, not `empty()`.

Reply:
ohh, fixed
too many problems with such a small fix)